pkg/cmd/taskrun/testdata/TestListTaskRuns-empty_list.golden

```diff
@@ -1 +1 @@
-No TaskRuns found
+No TaskRuns found
```

Comment:
Same as before, no taskruns found, should have a carriage return.. can you double check please?